src/main/kotlin/biokotlin/genome/MAFProcessingUtils.kt

```diff
@@ -24,7 +29,7 @@ import java.util.stream.Collectors
 // Data class to be used when creating a dataFrame for chrom percent coverage statistics
 // This may be used if can get Kotlin DataFrame vs Krangl DataFrame to work.
 data class ChromStats(val contig: String, val numRegionBPs: Int, val percentCov: Double, val percentId: Double)
-
+private val logger = KotlinLogging.logger {}
```

Comment:
Is this the logging we are using in BioKotlin? I see only one other file using that. src/main/kotlin/biokotlin/featureTree/Graph.kt

Reply:
@tcasstevens @zrm22 BioKotlin logging has never really been setup.  Jeffrey seems to have added the logger above, but this doesn't give us a specific class for these logging messages as we have with phg.  I think we need a BioKotlin Trello card for deciding on and setting up consistent logging.